pom.xml

```diff
@@ -15,8 +15,8 @@
     <url>https://wiki.jenkins.io/display/JENKINS/Extra+Tool+Installers+Plugin</url>
 
     <properties>
-        <jenkins.version>1.651.3</jenkins.version>
-        <java.level>7</java.level>
+        <jenkins.version>2.7.4</jenkins.version>
```

Comment:
If you switch to Java 8, please use 2.603

Reply:
Done.  Bumped to 2.60.3.